sidebars.js

```diff
@@ -20,7 +20,8 @@ module.exports = {
         'data-sources/mitomap',
         'data-sources/revel',
         'data-sources/splice-ai',
-        'data-sources/primate-ai'
+        'data-sources/primate-ai',
+		'data-sources/1000Genomes'
```

Comment:
Seems like you're using a different number of spaces.

Reply:
my editor was using a tab. Replaced them with spaces